packages/web/src/components/DatePicker/DatePicker.tsx

```diff
@@ -140,8 +140,9 @@ export const DatePicker: React.FC<Props> = ({
                 </ChangeDayButtonsStyledFlex>
 
                 {withTodayButton &&
-                  formattedCurrentMonth !== formattedSelectedMonth && (
+                //removed the condition here and put it in the prop 'isCurrentDate to check for truthyness
```

Comment:
The change looks good, but plz delete the comment so others who stumble upon this later don't get confused